packages/node-runtime-worker-thread/src/worker-runtime.ts

```diff
@@ -173,11 +175,11 @@ const workerRuntime: WorkerRuntime = {
     return serializeEvaluationResult(result);
   },
 
-  async getCompletions(code) {
```

Comment:
I removed some of these async keywords because linting complained that there is no await. I did double-check that the code does always return a Promise so it should be fine?

Reply:
I first tried to disable the lint rule, but then something else complained about the inverse (either vscode or the typescript compiler - can't remember right now), so it seemed easiest to just change the code.

Reply:
Maybe worth explicitly adding `Promise<value>` as a return type here just to make sure we don't accidentally break this in the future